core/src/main/scala/cats/Monad.scala

```diff
@@ -144,6 +144,15 @@ import scala.annotation.implicitNotFound
 
     tailRecM(branches.toList)(step)
   }
+
+  def replicateM[A](n: Int, fa: F[A]): F[List[A]] =
```

Comment:
for any lawful monad, these are exactly the same as replicateA aren't they? I am opposed to adding a function that is only different when the monad is not lawful.

Reply:
Yes, but it does not have the overhead of `traverseViaChain`/`Eval`/etc., if that would make a difference (would it?)

Reply:
People can override for efficiency but my opinion is that we should not have duplicative functions that are equivalent in general.

Secondly, generally I think we should not trade stack safety for performance since lack of stack safety makes the library randomly fail when sizes get a bit larger.

I think we should have it be as fast as possible that is also stack safe, IMO.

Reply:
Sounds good. Should I remove replicateM or just alias it?

Should I override replicateM with this version in StackSafeMonad?